uberfire-services/uberfire-services-backend/src/main/java/org/guvnor/common/services/backend/file/upload/AbstractFileServlet.java

```diff
@@ -226,30 +226,27 @@ private String processUpload(final FormData item,
     private String uploadFile(final FormData item,
                               final HttpServletRequest request,
                               final HttpServletResponse response) throws IOException {
-        final InputStream fileData = item.getFile().getInputStream();
         final org.uberfire.backend.vfs.Path targetPath = item.getTargetPath();
 
         if (!validateAccess(Paths.convert(targetPath),
                             response)) {
             return "FAIL";
         }
 
-        try {
-            switch (item.getOperation()) {
-                case CREATE:
-                    doCreate(targetPath,
-                             fileData,
-                             request,
-                             "Uploaded " + getTimestamp());
-                    break;
-                case UPDATE:
+        try (final InputStream fileData = item.getFile().getInputStream()) {
+            final FileOperation operation = item.getOperation();
+            if (operation == FileOperation.CREATE) {
+                doCreate(targetPath,
+                         fileData,
+                         request,
+                         "Uploaded " + getTimestamp());
+            } else if(operation == FileOperation.UPDATE) {
```

Comment:
@jomarko I would add a ` ` space after `if`

Reply:
yes, well spotted, changes applied, thank you